api_app/analyzers_manager/file_analyzers/doc_info.py

```diff
@@ -147,8 +157,35 @@ def run(self):
             MimeTypes.ZIP2.value,
         ]:
             results["follina"] = self.analyze_for_follina_cve()
+            results["uris"].extend(self.get_docx_urls())
+
+        results["extracted_CVEs"] = self.analyze_for_cve()
+        results["uris"].extend(self.get_external_relationships())
+        results["uris"].extend(self.extract_urls_from_IOCs())
+
+        results["uris"] = list(set(results["uris"]))  # uniq
+
         return results
 
+    def extract_urls_from_IOCs(self):
+        urls = []
+        try:
+            vbaparser = VBA_Parser(self.filepath)
+            analyzer_results = vbaparser.analyze_macros(True, True)
+        except Exception as e:
```

Comment:
I remember that you had to create a new instance of this due to a bug, right? If yes, insert a comment here explaining the reason why. Otherwise ,in future, it could be erronously refactored to avoid the double instance of the same class